http/src/main/resources/init-resources/jupyter-docker-compose-gce.yaml

```diff
@@ -14,7 +14,7 @@ services:
     network_mode: host
     volumes:
       # shared with welder
-      - "${NOTEBOOKS_DIR}":"${NOTEBOOKS_DIR}"
+      - "/work:${NOTEBOOKS_DIR}"
```

Comment:
Welder needs to have a shared volume with the Jupyter container because of its design -- it syncs notebooks on the filesystem with GCS. It's true that `welder.log` is placed in the same directory but I see that as a benefit as it gives debugging info to the user. I don't think there are any other welder-specific files that are shared (unless I'm forgetting something)..

Reply:
I can see `welder.log` be useful for power users.